src/primitives/block.cpp

```diff
@@ -11,58 +11,28 @@
 #include "utilstrencodings.h"
 #include "crypto/common.h"
 #include "crypto/scrypt.h"
+#include "chainparams.h"
```

Comment:
I'm not sure it is a good idea  to add dependence from chainparams

Reply:
\+